app/src/main/java/io/cgisca/godot/gpgs/PlayGameServicesGodot.kt

```diff
@@ -176,7 +177,7 @@ class PlayGameServicesGodot(godot: Godot) : GodotPlugin(godot), AchievementsList
         this.saveGameName = saveGameName
         val signInOptions = if (enableSaveGamesFunctionality) {
             val signInOptionsBuilder = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_GAMES_SIGN_IN)
-            signInOptionsBuilder.requestScopes(Drive.SCOPE_APPFOLDER).requestId()
```

Comment:
`.requestId()` should still be there though?

Reply:
For some reason if `.requestId()` is left there, it'll also cause the sign-in to get stuck.